docs/pages/versions/v37.0.0/sdk/status-bar.md

```diff
@@ -0,0 +1,150 @@
+title: StatusBar
+sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-37/packages/expo-status-bar'
+---
+
+import InstallSection from '~/components/plugins/InstallSection';
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+import SnackInline from '~/components/plugins/SnackInline';
+
+**`expo-status-bar`** gives you a component and imperative interface to control the app status bar to change its text color, background color, hide it, make it translucent or opaque, and apply animations to any of these changes. Exactly what you are able to do with the `StatusBar` component depends on the platform you're using.
+
+<PlatformsSection android emulator ios simulator web />
+
+> **Web browser support**: there is no API available on the web for controlling the operating system status bar, so `expo-status-bar` will noop (it will do nothing, it will also not error).
+
+## Installation
+
+<InstallSection packageName="expo-status-bar" />
+
+## Example Usage
+
+<SnackInline dependencies={["expo-status-bar"]}>
+
+```js
+import React from 'react';
+import { Text, View } from 'react-native';
+import { StatusBar } from 'expo-status-bar';
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <View
+        style={{
+          flex: 1,
+          backgroundColor: '#000',
+          alignItems: 'center',
+          justifyContent: 'center',
+        }}>
+        <Text style={{ color: '#fff' }}>Notice that the status bar has light text!</Text>
+        <StatusBar style="light" />
+      </View>
+    );
+  }
+}
+```
+
+</SnackInline>
+
+## API
+
+```js
+import { StatusBar } from 'expo-status-bar';
+```
+
+**[Components](#components)**
+
+- [`StatusBar`](#statusbar)
+
+**[Methods](#methods)**
+
+- [`setStatusBarBackgroundColor(backgroundColor, animated)`](#setstatusbarbackgroundcolorbackgroundcolor-animated)
+- [`setStatusBarHidden(hidden, animation)`](#setstatusbarhiddenhidden-animation)
+- [`setStatusBarNetworkActivityIndicatorVisible(visible)`](#setstatusbarnetworkactivityindicatorvisiblevisible)
+- [`setStatusBarStyle(style)`](#setstatusbarstylestyle)
+- [`setStatusBarTranslucent(translucent)`](#setstatusbartranslucenttranslucent)
+
+**[Types](#types)**
+
+- [`StatusBarAnimation`](#statusbaranimation)
+- [`StatusBarProps`](#statusbarprops)
+- [`StatusBarStyle`](#statusbarstyle)
+
+## Components
+
+### `StatusBar`
+
+A component that allows you to configure your status bar without directly calling imperative methods like `setBarStyle`.
+
+You will likely have multiple `StatusBar` components mounted in the same app at the same time. For example, if you have multiple screens in your app, you may end up using one per screen. The props of each `StatusBar` component will be merged in the order that they were mounted. This component is built on top of the [StatusBar](https://reactnative.dev/docs/statusbar) component exported from React Native, and it provides defaults that work better for Expo users.
```

Comment:
"Expo users" -> "Expo developers" (though also improves the UX for end users of Expo apps)